SwiftLabworkSecond/Controller/UserViewController.swift

```diff
@@ -0,0 +1,140 @@
+import UIKit
+
+
+// Добавила, чтобы отрисовывались странички университетов, при нажатии на них,
+// На сервис ссылку фото университета и его описание добавила
+class UserViewController: UIViewController {
+    private let config: ScreenConfig
+    private let mapper: BDUIMapperProtocol
+    private let networkService: NetworkServiceProtocol
+    
+    private let containerView = UIView()
```

Comment:
```suggestion
    private lazy var containerView = UIView()
```

Иначе у тебя контейнер загрузится раньше, чем основная вью) Ну и выкинет всю оптимизацию